lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -34,6 +34,8 @@
     - Installs packages using Chocolatey (U(http://chocolatey.org/)).
     - If Chocolatey is missing from the system, the module will install it.
     - List of packages can be found at U(http://chocolatey.org/packages).
+requirements:
+- chocolatey >= 0.10.4
```

Comment:
Is this actually required, what for?

Reply:
I thought it was required for the proxy stuff. And that's why I started adding things like `--no-progress`.
Just went through the Changelog and noticed `-r` actually requires v0.10.5 (which is not that much different).

Reply:
I updated it, need to test it with a playbook using the downgrade option :-)

Reply:
I've yet to make the proxy stuff, what is there works on `0.9` I believe but will have a PR later today to change that. Wasn't sure if there was anything that was already there that needed it.